frontend/src/pages/RegistrationUserRepresent/registrationUserRepresent.jsx

```diff
@@ -0,0 +1,135 @@
+import { useNavigate } from 'react-router-dom';
+import Panel, {
+    PanelBody,
+    PanelBodyTitle,
+    PanelNavigation,
+    PanelTitle,
+} from '../../components/Panel/panel';
+import Button from '../../components/Button/button';
+import Checkbox from '../../components/Checkbox/checkbox';
+import TextInput from '../../components/TextInput/textInput';
+import { useState } from 'react';
+import { Validator } from '../../utils/validation/validate';
+import { useAuthContext } from '../../provider/AuthProvider/authProvider';
+
+/**
+ * Registration page that asks the user to select
+ * who they represent: Company or Startup project
+ *
+ * @component
+ *
+ * @returns {JSX.Element}
+ */
+function RegistrationUserRepresent() {
+    // This component handles user registration
+    const navigate = useNavigate();
+
+    // Bind user to company
+    const { bindCompanyToUser } = useAuthContext();
+
+    // State to hold form data
+    const [formData, setFormData] = useState(
+        {
+            companyName: "",
+            representation: {
+                company: false,
+                startup: false
+            },
+            unexpected: ""
+        });
+
+    // State to hold validation errors
+    const [errors, setErrors] = useState({});
+
+    // Function to handle server-side errors
+    const handleError = (error) => {
+        // TODO
+        if (error.response && error.response.status === 401) {
+            setErrors(prev => ({
+                ...prev,
+                email: Validator.serverSideErrorMessages.companyAlreadyExist
+            }));
```

Comment:
Error message for company already exists is assigned to 'email' key, which is misleading. Assign the error to 'companyName' instead.
```
setErrors(prev => ({
    ...prev,
    companyName: Validator.serverSideErrorMessages.companyAlreadyExist
}));
```